Component/Pet/Pet.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+const StyledPet = styled.div`
+  display: flex;
+  border: 2px solid #000000;
+  border-radius: 10px;
+  margin: 10px;
+  padding: 10px;
+  justify-content: space-around;
+  align-items: center;
+  max-width: 250;
+  background-color: var(--secondary-color);
+  span {
+    font-size: 20vw;
+    position: absolute;
+  }
+  .portrait {
+    width: 45vw;
+    height: 30vh;
+    border-radius: 100%;
```

Comment:
If your intention is to create an ellipse, then border-radius: 100% is appropriate given the differing width and height. However, if you want a circle, you should use border-radius: 50% instead, as this will create a circle only if the width and height of the element are equal.